astropy/coordinates/builtin_frames/itrs.py

```diff
@@ -78,7 +78,11 @@ def earth_location(self):
         from astropy.coordinates.earth import EarthLocation
 
         cart = self.represent_as(CartesianRepresentation)
-        return EarthLocation(x=cart.x, y=cart.y, z=cart.z)
+        return EarthLocation(
```

Comment:
I now understand why you were wondering about my comment about adding `obstime` -- I changed the code a bit so that I would not have been confused...